modern/src/App.js

```diff
@@ -15,6 +15,8 @@ function useJsUpdates(node) {
   useEffect(() => node.js_listen("js_update", forceUpdate));
 }
 
+let mouseposition;
```

Comment:
Not sure how we want to do this. We need to keep track of the mouse position for some things, but the mousedown events get spammed, and I dont think the changes affect rendering. How do we want to store these?